base.cmake

```diff
@@ -246,26 +246,35 @@ ENDMACRO(SETUP_PROJECT_FINALIZE)
 #.rst:
 # .. ifmode:: user
 #
-#   .. command:: COMPUTE_PROJECT_ARGS
+#   .. command:: COMPUTE_PROJECT_ARGS (OUTPUT_VARIABLE [LANGUAGES <languages>...])
 #
 #     Compute the arguments to be passed to command PROJECT.
 #     For instance::
 #
-#       COMPUTE_PROJECT_ARGS(PROJECT_ARGS)
+#       COMPUTE_PROJECT_ARGS(PROJECT_ARGS LANGUAGES CXX)
 #       PROJECT(${PROJECT_NAME} ${PROJECT_ARGS})
 #
-#     :param VARIABLE: the variable where to write the result
+#     :param OUTPUT_VARIABLE: the variable where to write the result
+#     :param LANGUAGES: the project languages. It defaults to CXX.
 #
-MACRO(COMPUTE_PROJECT_ARGS VARIABLE)
+MACRO(COMPUTE_PROJECT_ARGS)
+  CMAKE_PARSE_ARGUMENTS(_project_ "" "VARIABLE" "LANGUAGES" ${ARGN})
```

Comment:
It should be `_project` here and not `_project_`

see [doc](https://cmake.org/cmake/help/latest/command/cmake_parse_arguments.html#command:cmake_parse_arguments)

> When done, cmake_parse_arguments will consider for each of the keywords listed in <options>, <one_value_keywords> and <multi_value_keywords> a variable composed of the given <prefix> followed by "_" and the name of the respective keyword

Reply:
Good catch. Thanks.